extension/src/fileSystem/tree.ts

```diff
@@ -89,7 +90,17 @@ export class TrackedExplorerTree
     return []
   }
 
-  public getTreeItem({ isDirectory, resourceUri }: PathItem): TreeItem {
+  public getTreeItem({ isDirectory, resourceUri, error }: PathItem): TreeItem {
+    if (error) {
+      const treeItem = new TreeItem(error.uri, TreeItemCollapsibleState.None)
+      treeItem.tooltip = getMarkdownString(`$(error) ${error.msg}`)
+      treeItem.iconPath = new ThemeIcon('blank')
+      treeItem.command = {
+        command: 'dvc.showOutput',
+        title: 'WAT'
```

Comment:
What does what mean in this context?

Reply:
It is literal garbage that I left in by mistake. `title` is a required field 😬.

Reply:
https://www.destroyallsoftware.com/talks/wat